src/wmnds/patterns/_patterns.scss

```
@@ -1,7 +1,7 @@
 // This file is imported from ./assets/sass/wmnds-components.scss
 
-@import "autocomplete/autocomplete"; // Autocomplete
-@import "banner/banner"; // Header styles
-@import "feedback-loop/feedback-loop"; // Feedback loop
-@import "footer/footer"; // Footer styles
-@import "header/header"; // Header styles
+@import 'autocomplete/autocomplete'; // Autocomplete
+@import 'banner/banner'; // Header styles
+@import 'feedback-loop/feedback-loop'; // Feedback loop
+@import 'footer/footer'; // Footer styles
+@import 'header/header'; // Header styles
```

Double quotes sass-lint error
```suggestion
@import "autocomplete/autocomplete"; // Autocomplete
@import "banner/banner"; // Header styles
@import "feedback-loop/feedback-loop"; // Feedback loop
@import "footer/footer"; // Footer styles
@import "header/header"; // Header styles
```